src/domains/printit-common/10_github_identity.tf

```diff
@@ -0,0 +1,116 @@
+data "azurerm_resource_group" "identity_rg" {
+  name = "${local.product}-identity-rg"
+}
+
+data "azurerm_kubernetes_cluster" "aks" {
+  name                = "${local.product}-${var.location_short}-${var.instance}-aks"
+  resource_group_name = "${local.product}-${var.location_short}-${var.instance}-aks-rg"
+}
+
+data "azurerm_key_vault" "key_vault" {
+  name                = "${local.product}-${var.location_short}-${var.domain}-kv"
+  resource_group_name = "${local.product}-${var.location_short}-${var.domain}-sec-rg"
+}
+
+# repos must be lower than 20 items
+locals {
+  repos_01 = [
+    "pagopa-print-payment-notice-service",
+    "pagopa-print-payment-notice-generator",
+  ]
+
+  federations_01 = [
+    for repo in local.repos_01 : {
+      repository = repo
+      subject    = var.env
+    }
+  ]
+
+  # to avoid subscription Contributor -> https://github.com/microsoft/azure-container-apps/issues/35
+  environment_cd_roles = {
+    subscription = [
+      "Contributor",
```

Comment:
Contributor is too much permissions for the subscription, we need to find a correct permission